cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -187,6 +187,17 @@ func generateFileContent(gen *protogen.Plugin, file *protogen.File, g *protogen.
 	}
 }
 
+// Copy comments from proto file to _grpc.pb.go file.
+func copyCommentsFromProto(g *protogen.GeneratedFile, service *protogen.Service) {
```

Comment:
suggestion: for the function, may be good to follow the similar convention as other functions in file starting from "gen" or "generate" like `genProtoComments` or `genRpcDefComments`

Reply:
Or `genServiceComments`?